drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -356,14 +361,14 @@ function DragAndDropTemplates(configuration) {
         );
     };
 
-    var sidebarButtonTemplate = function(buttonClass, iconClass, buttonText, disabled) {
+    var sidebarButtonTemplate = function(buttonClass, iconClass, buttonText, disabled, spinner) {
         return (
             h('span.sidebar-button-wrapper', {}, [
                 h(
                     'button.unbutton.btn-default.btn-small.'+buttonClass,
-                    {disabled: disabled || false, attributes: {tabindex: 0}},
+                    {disabled: disabled || spinner || false, attributes: {tabindex: 0}},
                     [
-                        h("span.btn-icon.fa."+iconClass, {attributes: {"aria-hidden": true}}, []),
+                        h("span.btn-icon.fa." + (spinner ? 'fa-spin.fa-spinner' : iconClass), {attributes: {"aria-hidden": true}}, []),
```

Comment:
@arbrandes This would be easier to read like this:

```js
        if (spinner) {
            iconClass = 'fa-spin.fa-spinner';
        }
        return (
            // ...
                        h(
                            "span.btn-icon.fa." + iconClass,
                            {attributes: {"aria-hidden": true}}, 
                            []
                        ),
                        // ...
```